src/EditorFeatures/Test2/IntelliSense/IntellisenseQuickInfoBuilderTests_Code.vb

```diff
@@ -0,0 +1,112 @@
+        Public Async Function QuickInfoForXmlCodeElement() As Task
+            Dim workspace =
+                <Workspace>
+                    <Project Language="C#" CommonReferences="true">
+                        <Document>
+                            class MyClass {
+                                /// &lt;summary&gt;
+                                /// Normalize    this, but &lt;c&gt;Not        this&lt;/c&gt;
+                                /// &lt;code&gt;
+                                /// line 1
+                                /// line     2
+                                /// &lt;/code&gt;
+                                /// Extra text after code.
+                                /// &lt;/summary&gt;
+                                void MyMethod() {
+                                    MyM$$ethod();
+                                }
+                            }
+                        </Document>
+                    </Project>
+                </Workspace>
+
+            Dim intellisenseQuickInfo = Await GetQuickInfoItemAsync(workspace, LanguageNames.CSharp)
+
+            Dim expected = New ContainerElement(
+                ContainerElementStyle.Stacked Or ContainerElementStyle.VerticalPadding,
+                New ContainerElement(
+                    ContainerElementStyle.Stacked,
+                    New ContainerElement(
+                        ContainerElementStyle.Wrapped,
+                        New ImageElement(New ImageId(KnownImageIds.ImageCatalogGuid, KnownImageIds.MethodPrivate)),
+                        New ClassifiedTextElement(
+                            New ClassifiedTextRun(ClassificationTypeNames.Keyword, "void"),
+                            New ClassifiedTextRun(ClassificationTypeNames.WhiteSpace, " "),
+                            New ClassifiedTextRun(ClassificationTypeNames.ClassName, "MyClass", navigationAction:=Sub() Return, "MyClass"),
+                            New ClassifiedTextRun(ClassificationTypeNames.Punctuation, "."),
+                            New ClassifiedTextRun(ClassificationTypeNames.MethodName, "MyMethod", navigationAction:=Sub() Return, "void MyClass.MyMethod()"),
+                            New ClassifiedTextRun(ClassificationTypeNames.Punctuation, "("),
+                            New ClassifiedTextRun(ClassificationTypeNames.Punctuation, ")"))),
+                   New ClassifiedTextElement(
+                       New ClassifiedTextRun(ClassificationTypeNames.Text, "Normalize this, but"),
+                        New ClassifiedTextRun(ClassificationTypeNames.WhiteSpace, " "),
+                        New ClassifiedTextRun(ClassificationTypeNames.Text, "Not        this", ClassifiedTextRunStyle.UseClassificationFont))),
+                New ClassifiedTextElement(
+                    New ClassifiedTextRun(ClassificationTypeNames.Text, "\r\nline 1\r\nline     2\r\n", ClassifiedTextRunStyle.UseClassificationFont)),
```

Comment:
💭  I would expect this to be rendered like `<br/>`

Example input:

https://github.com/dotnet/roslyn/blob/3e1cf13ade005df2497a4bf789381cb99024f5f8/src/EditorFeatures/Test2/IntelliSense/IntellisenseQuickInfoBuilderTests.vb#L326-L327

Example output:

https://github.com/dotnet/roslyn/blob/3e1cf13ade005df2497a4bf789381cb99024f5f8/src/EditorFeatures/Test2/IntelliSense/IntellisenseQuickInfoBuilderTests.vb#L371-L376

Reply:
I'll have to run the code to see if it looks the same with the current form.

Reply:
I'll add screenshots shortly.

Reply:
@sharwell Screenshot added.